### PR TITLE
Add batch creating prefabs from multiple selected actors in scene tree.

### PR DESCRIPTION
**Pull request by Tryibion**
Useful for adding many prefab actors at once when dragging from the scene. This does not allow renaming of each prefab on creating.

### CODE REVIEW

File: Source/Editor/Modules/PrefabsModule.cs [hunk 1]
```diff
@@ -37,6 +37,8 @@ public sealed class PrefabsModule : EditorModule
         /// </summary>
         public event Action<Prefab, Actor> PrefabApplied;
 
+        private Actor _actor;
```
Comment (mafiesto4) [inline Source/Editor/Modules/PrefabsModule.cs:40]: Could you add a comment that it's locally cached actor ref for prefab creation? And maybe rename it to something more self-explanatory.

File: Source/Editor/Content/GUI/ContentView.DragDrop.cs [hunk 2]
```diff
@@ -46,7 +46,14 @@ private bool ValidateDragActors(ActorNode actor)
         private void ImportActors(DragActors actors, ContentFolder location)
         {
             // Use only the first actor
```
Comment (mafiesto4) [inline Source/Editor/Content/GUI/ContentView.DragDrop.cs:48]: This comment can be removed as it's obsolete after this change

File: Source/Editor/Modules/PrefabsModule.cs [hunk 3]
```diff
@@ -107,29 +120,35 @@ private void OnPrefabCreated(ContentItem contentItem)
...
                 }
+                Undo.AddAction(new MultiUndoAction(actions));
+
+                _actor = null;
             }
 
             Editor.Instance.Windows.PropertiesWin.Presenter.BuildLayout();
         }
+        
+        private void GetActorsTree(List<Actor> list, Actor a)
```
Comment (mafiesto4) [inline Source/Editor/Modules/PrefabsModule.cs:143]: Could you move this function to `FlaxEditor.Utilities.Utils` as static. Then use it here, in `ActorStaticFlagsEditor` and `ActorLayerEditor` so we reuse more code.

**Review by mafiesto4 — CHANGES_REQUESTED** [review]

**Comment by Tryibion** [comment]
Updated to #1261

**Pull request closed without merging**